test/web-auth/web-auth.test.js

```diff
@@ -307,6 +317,75 @@ describe('auth0.WebAuth', function() {
         }
       ); // eslint-disable-line
     });
+    it('should return the id_token payload when there is no access_token', function(done) {
```

Comment:
shouldn't the payload also be returned when there's a VALID access token? That's the missing test case IMO, even if it's 95% similar to this